panic on command `add`

using command `add` will panic:

```sh
$ gm add -h
panic: unable to redefine 't' shorthand in "add" flagset: it's already used for "tags" flag
...
```

--- a/cmd/add/add.go
+++ b/cmd/add/add.go
@@ -47,7 +47,7 @@ func NewCmd(app *application.App) *cobra.Command {
 		},
 	}
 	c.Flags().StringVar(&app.Flags.Title, "title", "", "bookmark title")
-	c.Flags().StringVarP(&app.Flags.TagsStr, "tags", "t", "", "bookmark tags")
+	c.Flags().StringVar(&app.Flags.TagsStr, "tags", "", "bookmark tags")
 	cmdutil.HideFlag(c, "help")
 
 	return c

